tests/testapp/test_builtins.py

```diff
@@ -173,39 +187,46 @@ def setUp(self):
         self.condition_set = AppTodayConditionSet()
         self.server_dt = datetime.datetime(2016, 1, 1, 0, 0, 0)
         self.server_tz = ZoneInfo('America/Chicago')
-        self.server_dt_aware = self.server_dt.astimezone(self.server_tz)
```

Comment:
these timezone tests are really strangely written. normally astimezone would be called on an aware timezone (even aware with utc) so we know what its doing.

replace isn't the same thing as astimezone since it doesn't account for dst, and it is just a straight swap of tzinfo. Lucky these examples are in winter, otherwise you would be introducing a dst bug.

e.g. as timezone:

1st jan 2016 utc -> becomes
31st dec 2015 1800 chicago time

they are the same time, just in different zones. while yours:

1st jan 2016 utc -> replace
1st jan 2016 chicago

they are different times, offset by 6hrs.

However the tests never specified the original server_dt correctly as it was naive. In that case it uses your local system time (melbourne for me, warsaw for you).